tests/tests_runner.py

```diff
@@ -0,0 +1,68 @@
+import argparse
+import os
+from pathlib import Path
+from consts.resource_paths import nn_resource_path
+import sys
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-r", "--reruns", type=int, default=0)
+args = parser.parse_args()
+
+
+def print_result(name, success):
+    if success:
+        print(f"\033[1;34;40mTest {name}: \033[1;32;40mPassed!\033[0m")
```

Comment:
Add some global variables for these color codes, they are present in multiple lines, then use as ' f{'FLASHING_RED} .. '

Reply:
Added, also changed the way PrintColors are handled and parsed by `cli_print`